src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -16,6 +16,8 @@ import { PartRepository } from '../../data-access/repositories/interfaces/part-r
 import { Segment } from '../../model/entities/segment'
 import { SegmentRepository } from '../../data-access/repositories/interfaces/segment-repository'
 import { PieceRepository } from '../../data-access/repositories/interfaces/piece-repository'
+import {BasicRundown} from '../../model/entities/basic-rundown'
+import {AlreadyActivatedException} from '../../model/exceptions/already-activated-exception'
```

Comment:
Space before and after the brackets would be nice :) Such that it looks like this:
{ BasicRundown }
{ AlreadyActivatedException }